### ISSUE TITLE
Can't disable uim switching an input method on Shift+Space

### ISSUE THREAD

**Issue by Hi-Angel** [uim/uim#147]
I'm trying to disable input method getting toggled when hit <kbd>Shift</kbd>+<kbd>Space</kbd>. I have `(define enable-im-switch? #f)` in `~/.uim.d/customs/custom-global.scm` file, which I assume should disable IM getting switched.

And yet, if I run a gtk app as `GTK_IM_MODULE=uim ./a`, and press <kbd>Shift</kbd>+<kbd>Space</kbd>, IM gets switched to russian.

I found this as part of debugging of [this problem](https://bugs.kde.org/show_bug.cgi?id=408855) *(which as far as I can tell is only or mostly caused by uim being used by gtk apps)*.

## Additional information

uim version 1.8.8
Content of `~/.uim.d/customs/custom-global.scm`:

```scheme
(define custom-activate-default-im-name? #f)
(define custom-preserved-default-im-name 'm17n-en-ispell)
(define default-im-name #f)
(define enabled-im-list '(anthy anthy-utf8 skk tutcode byeoru latin elatin m17n-ru-kbd m17n-ru-phonetic m17n-ru-translit m17n-ru-yawerty m17n-sa-harvard-kyoto m17n-sa-IAST m17n-sa-itrans m17n-sd-inscript m17n-si-phonetic-dynamic m17n-si-samanala m17n-si-singlish m17n-si-sumihiri m17n-si-transliteration m17n-si-wijesekera m17n-sk-kbd m17n-sr-kbd m17n-sv-post m17n-ta-inscript m17n-ta-itrans m17n-ta-lk-renganathan m17n-ta-phonetic m17n-ta-tamil99 m17n-ta-typewriter m17n-ta-vutam m17n-tai-sonla-kbd m17n-te-apple m17n-te-inscript m17n-te-itrans m17n-te-pothana m17n-te-rts m17n-te-sarala m17n-th-kesmanee m17n-th-pattachote m17n-th-tis820 m17n-ug-kbd m17n-uk-kbd m17n-ur-phonetic m17n-uz-kbd m17n-vi-han m17n-vi-nomvni m17n-vi-nomtelex m17n-vi-tcvn m17n-vi-telex m17n-vi-viqr m17n-vi-vni m17n-yi-yivo m17n-zh-bopomofo m17n-zh-cangjie m17n-zh-pinyin-vi m17n-zh-pinyin m17n-zh-py-b5 m17n-zh-py-gb m17n-zh-py m17n-zh-quick m17n-zh-tonepy-b5 m17n-zh-tonepy-gb m17n-zh-tonepy m17n-zh-zhuyin m17n-am-sera m17n-ar-kbd m17n-ar-translit m17n-as-inscript m17n-as-itrans m17n-as-phonetic m17n-ath-phonetic m17n-be-kbd m17n-bla-phonetic m17n-bn-disha m17n-bn-inscript m17n-bn-itrans m17n-bn-probhat m17n-bn-unijoy m17n-bo-ewts m17n-bo-tcrc m17n-bo-wylie m17n-cmc-kbd m17n-cr-western m17n-cs-kbd m17n-da-post m17n-dv-phonetic m17n-el-kbd m17n-eo-h-fundamente m17n-eo-h-sistemo m17n-eo-plena m17n-eo-q-sistemo m17n-eo-vi-sistemo m17n-eo-x-sistemo m17n-fa-isiri m17n-fr-azerty m17n-grc-mizuochi m17n-gu-inscript m17n-gu-itrans m17n-gu-phonetic m17n-he-kbd m17n-hi-inscript m17n-hi-itrans m17n-hi-optitransv2 m17n-hi-phonetic m17n-hi-remington m17n-hi-typewriter m17n-hi-vedmata m17n-hr-kbd m17n-hy-kbd m17n-ii-phonetic m17n-en-ispell m17n-iu-phonetic m17n-ja-anthy m17n-ja-tcode m17n-ja-trycode m17n-ka-kbd m17n-kk-arabic m17n-kk-kbd m17n-km-yannis m17n-kn-inscript m17n-kn-itrans m17n-kn-kgp m17n-kn-optitransv2 m17n-kn-typewriter m17n-ko-han2 m17n-ko-romaja m17n-ks-inscript m17n-ks-kbd m17n-lo-kbd m17n-lo-lrt m17n-mai-inscript m17n-rfc1345 m17n-ssymbol m17n-syrc-phonetic m17n-unicode m17n-latn-post m17n-latn-pre m17n-latn1-pre m17n-lsymbol m17n-math-latex m17n-ml-enhanced-inscript m17n-ml-inscript m17n-ml-itrans m17n-ml-mozhi m17n-ml-remington m17n-ml-swanalekha m17n-mr-inscript m17n-mr-itrans m17n-mr-phonetic m17n-mr-remington m17n-mr-typewriter m17n-my-kbd m17n-ne-rom-translit m17n-ne-rom m17n-ne-trad-ttf m17n-ne-trad m17n-nsk-phonetic m17n-oj-phonetic m17n-or-inscript m17n-or-itrans m17n-or-phonetic m17n-pa-anmollipi m17n-pa-inscript m17n-pa-itrans m17n-pa-jhelum m17n-pa-phonetic m17n-ps-phonetic zm wb86 py pyunihan pinyin-big5 viqr ipa-x-sampa look ajax-ime google-cgiapi-jp baidu-olime-jp))
(define enable-im-switch? #f)
(define switch-im-key '("<Control>Shift_key" "<Shift>Control_key"))
(define switch-im-key? (make-key-predicate '("<Control>Shift_key" "<Shift>Control_key")))
(define switch-im-skip-direct-im? #f)
(define enable-im-toggle? #f)
(define toggle-im-key '("<Meta> "))
(define toggle-im-key? (make-key-predicate '("<Meta> ")))
(define toggle-im-alt-im 'direct)
(define uim-color 'uim-color-uim)
(define candidate-window-style 'vertical)
(define candidate-window-position 'caret)
(define enable-lazy-loading? #t)
(define bridge-show-input-state? #f)
(define bridge-show-with? 'mode)
(define bridge-show-input-state-time-length 3)
```
setxkbmap -query:
```haskell
rules:      evdev
model:      pc105
layout:     us,ru
options:    grp:rctrl_rshift_toggle,compose:rwin,grp_led:scroll
```

**Comment by Hi-Angel** [comment]
Omg, I finally found it: I had to open preferences *(e.g. `uim-pref-gtk3` or `uim-pref-qt`)*, go to "Global key bindings 1" page, and there it was, the `[Global] on` and `[Global] off` fields have had the Shift+Space key. I removed it there *(and also from "Look keybindings")*, and now lang no longer gets switched.

The main question that remains: where were these settings stored? I haven't found word "space" *(any case)* anywhere at `~/.uim.d/`, so apparently it stores that somewhere else.
